microsoft/PowerToys
# KBM Editor tests

## Summary of the Pull Request

**What is this about:**
- Move almost all files from the Editor into the static library in order to test. 
- Use `Generated Files\resource.h` from the edior

**What is include in the PR:** 

**How does someone test / validate:** 
- Verify that both editor windows works
- Run editor's tests
- Verify labels

## Quality Checklist


## Contributor License Agreement (CLA)
A CLA must be signed. If not, go over [here](https://cla.opensource.microsoft.com/microsoft/PowerToys) and sign the CLA.
